src/node/nodes.h

```diff
@@ -17,12 +17,14 @@ namespace ccf
   enum class NodeStatus
   {
     PENDING = 0,
-    TRUSTED = 1,
-    RETIRED = 2
+    CATCHING_UP = 1,
+    TRUSTED = 2,
+    RETIRED = 3
```

Comment:
If NodeStatus is serialized anywhere and sent as a network msg or put in the ledger reordering the numbers like this will break back compat